src/workflows-app/SubmissionConfig.test.js

```diff
@@ -1,19 +1,26 @@
-import { render, screen, waitFor } from '@testing-library/react';
+import { act, fireEvent, render, screen, within } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
-import { act } from 'react-dom/test-utils';
 import { h } from 'react-hyperscript-helpers';
 import { Ajax } from 'src/libs/ajax';
 import { getConfig } from 'src/libs/config';
 import * as Nav from 'src/libs/nav';
 import { AppProxyUrlStatus, getUser, workflowsAppStore } from 'src/libs/state';
+import { SelectHelper } from 'src/testing/test-utils';
```

Comment:
@nawatts should this import be `@terra-ui-packages/test-utils` due to #4090 or stay unchanged?

Reply:
This should not change. #4090 only moved Jest configuration to `@terra-ui-packages/test-utils`.